scargo/file_generators/templates/conan/profile.j2

```diff
@@ -0,0 +1,64 @@
+[env]
+{% if config.project.target.family == "stm32" %}
+CC=/opt/gcc-arm-none-eabi/bin/arm-none-eabi-gcc
+CXX=/opt/gcc-arm-none-eabi/bin/arm-none-eabi-g++
+{% elif config.project.target.family == "esp32" %}
+CC=/opt/esp-idf/tools/xtensa-esp32-elf/esp-2021r2-patch5-8.4.0/xtensa-esp32-elf/bin/xtensa-esp32-elf-gcc
```

Comment:
fix path, can we take it from environmental, as it will crash with idf update

Reply:
Issue created for checking native paths:
https://github.com/Spyro-Soft/scargo/issues/275